test/libsolidity/SolidityExpressionCompiler.cpp

```diff
@@ -153,7 +153,7 @@ bytes compileFirstExpression(
 					parametersSize--
 				);
 
-			ExpressionCompiler(context).compile(*extractor.expression());
+			ExpressionCompiler(context, dev::test::Options::get().optimize).compile(*extractor.expression());
```

Comment:
The reason for these tests to change was that we did not take the global test parameter into account before. I think this is a good test, because it shows the effect of `order literals` optimization.

Reply:
But all the deployment sizes increased.

Reply:
No, the non-optimized expectations are unchanged.